integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciExtension.java

```diff
@@ -121,28 +124,46 @@ private OciExtension() {
     }
 
     /**
-     * Returns the {@link OciConfig} that is currently defined in the bootstrap environment. If one is not defined under
-     * config {@value OciConfig#CONFIG_KEY} then a default implementation will be constructed.
+     * Returns the global {@link OciConfig} that is currently defined in the bootstrap environment.
+     * <p>
+     * The implementation will first look for an {@code oci.yaml} file, and if found will use that file to establish the global
+     * oci-specific bootstrap {@link io.helidon.config.spi.ConfigSource}.
+     * <p>
+     * If the implementation is unable to find this file, then a fallback mechanism will be used to find it in the configuration
+     * found in the {@link InjectionServices#globalBootstrap()}, using a top-level attribute key named
+     * {@value OciConfigBlueprint#CONFIG_KEY}.
+     * <p>
+     * The final fallback mechanism will use an {@code auto} authentication strategy - see {@link OciConfigBlueprint} for details.
      *
      * @return the bootstrap oci config bean
+     * @see OciConfigBlueprint
      */
     public static OciConfig ociConfig() {
-        Optional<Bootstrap> bootstrap = InjectionServices.globalBootstrap();
-        if (bootstrap.isEmpty()) {
-            LOGGER.log(System.Logger.Level.DEBUG, "No bootstrap - using default oci config");
-            return DEFAULT_OCI_CONFIG_BEAN.get();
-        }
+        // we do it this way to allow for the possibility of system and env vars to be used for the auth-strategy definition
+        // (not advertised in the javadoc)
+        io.helidon.common.config.Config config = Config.create(
```

Comment:
This is the correct way, the `::build` is useless:
```
Config config = Config.create(ConfigSources.classpath(OCI_GLOBAL_CONFIG_FILE).optional(),
                      ConfigSources.file(OCI_GLOBAL_CONFIG_FILE).optional());
```

Reply:
addressed and pushed.